Policies/github-additional-product-terms.md

```diff
@@ -1,72 +1,31 @@
 ---
-title: GitHub Additional Product Terms
+title: GitHub Terms for Additional Products and Features
+redirect_from:
+  - /github/site-policy/github-additional-product-terms
 versions:
   free-pro-team: '*'
 topics:
-  - policy
-  - legal
+  - Policy
+  - Legal
 ---
 
-Version Effective Date: February 22, 2021
+Version Effective Date: May 26, 2021
 
-When you create an Account, you're given access to lots of different features and products that are all a part of the Service. Because many of these features and products offer different functionality, they may require additional terms and conditions specific to that feature or product. Below, we've listed those features and products, along with the corresponding additional terms that apply to your use of them.
+When you use GitHub, you may be given access to lots of additional products and features ("Additional Products and Features"). Because many of the Additional Products and Features offer different functionality, specific terms for that product or feature may apply in addition to your main agreement with us—the GitHub Terms of Service, GitHub Corporate Terms of Service, GitHub General Terms, or Microsoft volume licensing agreement (each, the "Agreement"). Below, we've listed those products and features, along with the corresponding additional terms that apply to your use of them.
```

Comment:
What is the, "Microsoft volume licensing agreement"?

Are all users required to agree to the, "Microsoft volume licensing agreement"?

If so, could you please provide a link to that agreement?

Reply:
@sickcodes the Microsoft volume licensing agreement is for Enterprise customers. 
>Are all users required to agree to the, "Microsoft volume licensing agreement"?

No - if you have a GitHub account, you've agreed to one of these (there's an "or" there)
>your main agreement with us—the GitHub Terms of Service, GitHub Corporate Terms of Service, GitHub General Terms, or Microsoft volume licensing agreement (each, the "Agreement")

so we are saying you this doc provides terms that are additional to whichever one of those is your main agreement.